template_params-rails.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "template_params", "~> 0.1.0"
+  spec.add_runtime_dependency "template_params", ">= 0.1.0", "< 0.3"
```

Comment:
That's odd. I've definitely used an array here before

```
[">= 0.1.0", "< 0.3"]
```

.. but the docs only show multiple arguments, 

```
">= 0.1.0", "< 0.3"
```

So I guess that's fine.